includes/abstracts/abstract-wc-shipping-method.php

```diff
@@ -168,7 +168,7 @@ public function calculate_shipping( $package = array() ) {}
 	 * @return boolean
 	 */
 	public function is_taxable() {
-		return wc_tax_enabled() && 'taxable' === $this->tax_status && ! WC()->customer->get_is_vat_exempt();
+		return wc_tax_enabled() && 'taxable' === $this->tax_status && !WC()->customer && ! WC()->customer->get_is_vat_exempt();
```

Comment:
This logic does not look correct. `! WC()->customer` is true when there is no customer, so `WC()->customer->get_is_vat_exempt()` will cause an error.

The underlying issue seems to be that you are likely doing things before the `woocommerce_init` action, as [`WC()->customer` will always exist on the site frontend](https://github.com/woocommerce/woocommerce/blob/master/includes/class-woocommerce.php#L489).